com.github.neithern.g4music.json

```diff
@@ -0,0 +1,60 @@
+{
+    "app-id": "com.github.neithern.g4music",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "g4music",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-music:ro",
+        "--filesystem=xdg-download:ro"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "g4music",
+            "builddir": true,
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "com.github.neithern.g4music.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "com.github.neithern.g4music.appdata.xml"
+                },
```

Comment:
there is a .desktop and a .appdata upstream. So I don't see why you need this here.

Reply:
removed